models/channel/src/channel-draft/presets/index.ts

```diff
@@ -1,10 +1,12 @@
+import SampleDataFashion from './sample-data-fashion';
 import withInventorySupplyAndProductDistributionRoles from './with-inventory-supply-and-product-distribution-roles';
 import withInventorySupplyAndProductDistributionRolesNoAddress from './with-inventory-supply-and-product-distribution-roles-no-address';
 import withInventorySupplyRole from './with-inventory-supply-role';
 import withInventorySupplyRoleNoAddress from './with-inventory-supply-role-no-address';
 import withProductDistributionRole from './with-product-distribution-role';
 
 const presets = {
+  SampleDataFashion,
```

Comment:
Nit: lowercase for consistency? `sampleDataFashion`

Reply:
[Sounds good!](https://github.com/commercetools/test-data/pull/205/commits/cd126a4c140b9ae7a53d78fb295079ec14cb73d1)